docs/index.md

```diff
@@ -17,5 +17,14 @@ This document specifies the future protocol for diaspora. diaspora\* release 0.6
 [entities][entities] with this protocol, but still sends entities with an older protocol. Starting with diaspora\*
 release X this protocol is fully supported.
 
+## Implementations
+
+An implementation of this protocol is available as a Ruby Gem under the AGPL [on Github][github]. This is the library used by the diaspora* project.
+
+The [Friendica][friendica] project also has [its implementation in PHP][phpimplementation].
+
 [communication]: https://wiki.diasporafoundation.org/How_we_communicate
 [entities]: {{ site.baseurl }}/entities/
```

Comment:
Yes, the docs are rendered on [github pages](https://pages.github.com/), you can test this locally with `gem install jekyll` and `jekyll serve`.